pretenders/common/http.py

```diff
@@ -41,10 +42,16 @@ def serialize(self):
         }
         return json.dumps(data)
 
-
 def binary_to_ascii(data):
-    return base64.b64encode(data).decode('ascii')
+    python3 = sys.version_info.major >= 3
+
+    if python3 and isinstance(data, str):
+        input_bytes = data.encode('utf8')
```

Comment:
This is quite a major assumption, that if data is a string it will be utf-8 encoded.
I would prefer not to make these assumptions, and have input data as an unambiguous byte string.